src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2379,7 +2379,7 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
 
         network_observability = self.context.get_enable_network_observability()
         if network_observability is not None:
-            network_profile.network_monitoring = self.models.NetworkMonitoring(
```

Comment:
Queued a [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=74879487&view=results) to validate the change.

Reply:
Test passed.